CMSgov/design-system
# [WNMGDS-2260] Updated paths for preact/web component resources

## Summary

Looks like the paths for web components and preact components were updated? In any event this updates the cdn-index creation script to update to those paths.

## How to test

1. `yarn clean && yarn install && yarn build` .. check output of `dist/index.html`, should reflect the correct paths

## Checklist
